content/getting-started/php.textile

```diff
@@ -0,0 +1,215 @@
+* Retrieve the presence set of the channel.
+* Retrieve the messages you sent in the guide from history.
+
+<aside data-type='note'>
+<p>This guide covers the Ably PHP SDK which provides RESTful functionality for publishing messages, retrieving history, and checking presence. If you need realtime features like subscribing to channels and receiving live updates, check out the "Ably Laravel broadcaster":https://github.com/ably/laravel-broadcaster which integrates with Laravel's broadcasting system.</p>
+</aside>
+
+h2(#prerequisites). Prerequisites
+
+* "Sign up":https://ably.com/signup for an Ably account.
+** Create a new app, and create your first API key.
+** Your API key will need the @publish@, @presence@, and @history@ capabilities.
+
+* Install the Ably CLI:
+
+```[sh]
+npm install -g @ably/cli
+```
+
+* Run the following to log in to your Ably account and set the default app and API key:
+
+```[sh]
+ably login
+
+ably apps switch
+ably auth keys switch
+```
+
+* Install "PHP":https://www.php.net/downloads.php version 7.2 or greater.
+* Create a new project and install the Ably Pub/Sub PHP SDK:
+
+```[sh]
+# Create a new directory for your project
+mkdir ably-php-quickstart
+cd ably-php-quickstart
+
+# Initialize Composer (if you don't have a composer.json yet)
+composer init
+
+# Install the Ably PHP SDK
+composer require ably/ably-php
+```
+
+<aside data-type='note'>
+<p>The code examples in this guide include a demo API key. If you wish to interact with the Ably CLI and view outputs within your Ably account, ensure that you replace them with your own API key.</p>
+</aside>
+
+
+h2(#step-1). Step 1: Configure the Ably Rest SDK client
+
+Clients can make REST API requests to Ably when they instantiate a REST SDK instance. This enables them to publish messages to channels and retrieve channel information.
+
+* Create a @get_started.php@ file in your project and add the following code to instantiate the SDK and make requests to Ably. At the minimum you need to provide an authentication mechanism. Using an API key directly is fine for server-side environments like PHP where the key remains secure. However, for frontend clients, you should use token authentication instead of exposing API keys. In this example, the @clientId@ will be the name of the client sending the message:
+
+```[php]
+<?php
+
+require 'vendor/autoload.php';
+
+use Ably\AblyRest;
+
+$ably = new AblyRest(['key' => '{{API_KEY}}', 'clientId' => 'my-first-client']);
+```
+
+h2(#step-2). Step 2: Generate a JWT
+
+The Ably PHP SDK provides token authentication for secure client connections. Authentication with a JWT is recommended over using API keys directly in production because they can be scoped to specific capabilities and have expiration times. This is essential when authenticating frontend clients where embedding API keys would expose them publicly. With PHP the API key would be considered securely stored as it is on the server side.
```

Comment:
```suggestion
The Ably PHP SDK provides token authentication for secure client connections. Authentication with a JWT is recommended over using API keys directly in production because they can be scoped to specific capabilities and have expiration times. This is essential when authenticating frontend clients where embedding API keys would expose them publicly. With PHP the API key would be considered secure as it's stored on the server.
```